src/components/Header/Navigation/index.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from "react";
+
+class HeaderNavigation extends Component {
+  render() {
+    return <React.Fragment>This is the Navigation.</React.Fragment>;
```

Comment:
You can switch this to <Fragment> and call it as `import React, { Component, Fragment } from 'react';` and it'll achieve the same thing. A little nitpicky but just a helpful suggestion!